pkg/client/injection/reconciler/samples/v1alpha1/addressableservice/stub/controller.go

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Should we be checking in these stubs?

Reply:
Yes, they are intended to be for reference for the integration.